Client/Client/RegisterPage.xaml.cs

```diff
@@ -0,0 +1,54 @@
+using Client.ViewModels;
+using System.Text.RegularExpressions;
+
+namespace Client;
+
+public partial class RegisterPage : ContentPage
+{
+	public RegisterPage(RegisterViewModel vm)
+	{
+		InitializeComponent();
+		BindingContext = vm;
```

Comment:
Add line "Shell.SetTabBarIsVisible(this, false);"  To remove navigation pane.

Reply:
Solved